builtin/game/misc.lua

```diff
@@ -145,7 +145,7 @@ end
 
 function core.get_item_group(name, group)
 	if not core.registered_items[name] or not
-			core.registered_items[name].groups[group] then
+	core.registered_items[name].groups[group] then
```

Comment:
```lua
if ...
<two tabs> ... then
<one tab> <body>
end
```
is better. See https://dev.minetest.net/Lua_code_style_guidelines#Continuation_lines.